src/Sylius/Behat/Page/Admin/SlideshowBlock/CreatePage.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\SlideshowBlock;
+
+use Behat\Mink\Element\NodeElement;
+use Sylius\Behat\Page\Admin\Crud\CreatePage as BaseCreatePage;
+
+/**
+ * @author Vidy Videni <vidy.videni@gmail.com>
+ */
+class CreatePage extends BaseCreatePage implements CreatePageInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function setTitle($title)
+    {
+        $this->getSession()->getPage()->fillField('Title', $title);
```

Comment:
``$this->getDocument()->fillField(...)`` or ``$this->getElement('title')->setValue(...)``.